pkg/reconciler/events/cloudevent/cloudevent_test.go

```diff
@@ -83,6 +86,34 @@ func getPipelineRunByCondition(status corev1.ConditionStatus, reason string) *v1
 	}
 }
 
+func getRunByCondition(status corev1.ConditionStatus, reason string) *v1alpha1.Run {
```

Comment:
nit: a clearer name might be `createRunWithCondition`

Reply:
👍